pkg/prometheus/promcfg.go

```diff
@@ -1122,6 +1134,16 @@ func getSampleLimit(user uint64, enforced *uint64) uint64 {
 	return user
 }
 
+func getTargetLimit(user uint64, enforced *uint64) uint64 {
```

Comment:
this can be merged with `getSampleLimit()` above (maybe renamed `getLimit()`)

Reply:
Done